core/payment-driver/zksync/src/driver.rs

```diff
@@ -209,9 +210,9 @@ impl PaymentDriver for ZksyncDriver {
             .map_err(GenericError::new)??;
 
         let mode = msg.mode();
-        let network = DEFAULT_NETWORK; // TODO: Implement multi-network support
-        let token = DEFAULT_TOKEN; // TODO: Implement multi-network support
-        bus::register_account(self, &address, network, token, mode).await?;
+        let network = msg.network().unwrap_or(DEFAULT_NETWORK.to_string());
+        let token = msg.token().unwrap_or(DEFAULT_TOKEN.to_string());
```

Comment:
It should be the default token **for this network** not universally-default token. If `msg.network` is `"mainnet"` and `msg.token` is `None` then `token` should be `"GLM"` not `"tGLM"`.

Please check if you didn't make the same error somewhere else.

Reply:
was replacing all `DEFAULT_TOKEN` with `get_network_token()` yesterday, will fix this and check for the other locations